README.md

```diff
@@ -6,51 +6,49 @@ Visualize Chaos Mesh Events with Grafana.
 
 ## Usage
 
-For detailed usage, please check [Tutorial](https://github.com/chaos-mesh/chaos-mesh-datasource/blob/master/docs/usage.md).
+### Configure the data source
 
-## Development
+Go to Grafana -> Configuration -> Data Sources, click `Add data source` button.
+
+![add-data-source](docs/assets/add-datasource.png)
```

Comment:
This is not true, please update all the docs to link to full path. Relative path is not okay to display in the Grafana. For example, `https://github.com/chaos-mesh/chaos-mesh-datasource/blob/master/docs/usage.md`